web/translation/translate.fa_IR.toml

```diff
@@ -213,151 +213,151 @@
-"FreedomStrategy" = "روش استفاده از شبکه خروجی مستقیم"
-"FreedomStrategyDesc" = "تعیین روش استفاده از خروجی برای پرتکل مستقیم"
-"RoutingStrategy" = "پیکربندی استراتژی حل دامنه در مسیریابی"
-"RoutingStrategyDesc" = "تعیین استراتژی مسیریابی کلی برای پیدا کردن دامنه"
-"Torrent" = "فیلتر کردن بیت تورنت"
-"TorrentDesc" = "الگوی تنظیمات را برای فیلتر کردن پروتکل بیت تورنت برای کاربران تغییر میدهد"
-"PrivateIp" = "جلوگیری از اتصال آیپی های خصوصی یا محلی"
-"PrivateIpDesc" = "الگوی تنظیمات را برای فیلتر کردن اتصال آیپی های خصوصی یا محلی و بسته های سرگردان تغییر میدهد"
-"Ads" = "مسدود کردن تبلیغات"
-"AdsDesc" = "الگوی تنظیمات را برای مسدود کردن تبلیغات تغییر میدهد"
-"Family" = "فعال کردن حالت خانواده"
-"FamilyDesc" = "برای جلوگیری از ارتباط با وبسایت های ناامن"
-"IRIp" = "جلوگیری از اتصال آیپی های ایران"
-"IRIpDesc" = "الگوی تنظیمات را برای فیلتر کردن اتصال آیپی های ایران تغییر میدهد"
-"IRDomain" = "جلوگیری از اتصال دامنه های ایران"
-"IRDomainDesc" = "الگوی تنظیمات را برای فیلتر کردن اتصال دامنه های ایران تغییر میدهد"
-"ChinaIp" = "جلوگیری از اتصال آیپی های چین"
-"ChinaIpDesc" = "الگوی تنظیمات را برای فیلتر کردن اتصال آیپی های چین تغییر میدهد"
-"ChinaDomain" = "جلوگیری از اتصال دامنه های چین"
-"ChinaDomainDesc" = "الگوی تنظیمات را برای فیلتر کردن اتصال دامنه های چین تغییر میدهد"
-"RussiaIp" = "جلوگیری از اتصال آیپی های روسیه"
-"RussiaIpDesc" = "الگوی تنظیمات را برای فیلتر کردن اتصال آیپی های روسیه تغییر میدهد"
-"RussiaDomain" = "جلوگیری از اتصال دامنه های روسیه"
-"RussiaDomainDesc" = "الگوی تنظیمات را برای فیلتر کردن اتصال دامنه های روسیه تغییر میدهد"
-"DirectIRIp" = "ارتباط مستقیم به آیپی های ایران"
-"DirectIRIpDesc" = "الگوی تنظیمات را برای ارتباط مستقیم به آیپی های ایران تغییر میدهد"
-"DirectIRDomain" = "ارتباط مستقیم به دامنه های ایران"
-"DirectIRDomainDesc" = "الگوی تنظیمات را برای ارتباط مستقیم به دامنه های ایران تغییر میدهد"
-"DirectChinaIp" = "ارتباط مستقیم به آیپی های چین"
-"DirectChinaIpDesc" = "الگوی تنظیمات را برای ارتباط مستقیم به آیپی های چین تغییر میدهد"
-"DirectChinaDomain" = "ارتباط مستقیم به دامنه های چین"
-"DirectChinaDomainDesc" = "الگوی تنظیمات را برای ارتباط مستقیم به دامنه های چین تغییر میدهد"
-"DirectRussiaIp" = "ارتباط مستقیم به آیپی های روسیه"
-"DirectRussiaIpDesc" = "الگوی تنظیمات را برای ارتباط مستقیم به آیپی های روسیه تغییر میدهد"
-"DirectRussiaDomain" = "ارتباط مستقیم به دامنه های روسیه"
-"DirectRussiaDomainDesc" = "الگوی تنظیمات را برای ارتباط مستقیم به دامنه های روسیه تغییر میدهد"
-"GoogleIPv4" = "استفاده از آیپی ورژن 4 برای اتصال به گوگل"
-"GoogleIPv4Desc" = "مسیردهی جدید برای اتصال به گوگل با آیپی ورژن 4 اضافه میکند"
-"NetflixIPv4" = "استفاده از آیپی ورژن 4 برای اتصال به نتفلیکس"
-"NetflixIPv4Desc" = "مسیردهی جدید برای اتصال به نتفلیکس با آیپی ورژن 4 اضافه میکند"
+"basicTemplate" = "الگو پایه"
+"advancedTemplate" = "الگو پیشرفته"
+"generalConfigs" = "استراتژی مسیریابی"
+"generalConfigsDesc" = "این گزینه‌هااستراتژی کلی مسیریابی ترافیک را بازتاب می‌دهد"

```

Comment:
It is not about only routing strategies.
First option is only for freedom setting. Please revert it back